lib/timeutils/unixtime.c

```diff
@@ -90,6 +90,8 @@ unix_time_set_from_normalized_wall_clock_time_with_tz_hint(UnixTime *self, WallC
                  - target_gmtoff;
 
   self->ut_gmtoff = target_gmtoff;
+  wct->wct_gmtoff = self->ut_gmtoff;
```

Comment:
This function is like an assignment for me. But shouldn't we consider wct as const in this context? It looks very strange that the assignee is assigned back to the assigner.

Reply:
btw: I wrote a comment in the unixtime.h header as well, so it's not just a commit message.

Reply:
with the new convert_XXX() functions this should be even more appearent, so resolving this conversation, reopen if you disagree.